webcompat/static/js/lib/wizard/ui-utils.js

```diff
@@ -17,11 +17,8 @@ export const scrollToElement = (element, delay = 250) => {
   setTimeout(() => {
     const topOfElement =
       window.pageYOffset + element.getBoundingClientRect().top - HEADER_HEIGHT;
-    window.scroll({ top: topOfElement, behavior: "smooth" });
+    $("html, body").animate({ scrollTop: topOfElement });
   }, delay);
-
-  //@todo figure out why this is needed
-  //this.isSubproblem ? 450 : 250
```

Comment:
I've accounted for this todo above, by putting showContainer in a callback of `show` function, that way we don't need a bigger timeout for animating the subcategory

Reply:
thanks, i was wondering what https://github.com/webcompat/webcompat.com/pull/3390/files#diff-1d0db961f9ba9046d10b32edb5d31394R16 was for.